tests/test_summary.py

```diff
@@ -942,7 +942,10 @@ def test_smry_meta_synthetic():
 def test_fix_dframe_for_libecl(dframe, expected_dframe):
     """Test the dataframe preprocessor/validator for df2eclsum works"""
     pd.testing.assert_frame_equal(
-        _fix_dframe_for_libecl(dframe), expected_dframe, check_index_type=False
+        _fix_dframe_for_libecl(dframe),
+        expected_dframe,
+        check_index_type=False,
+        check_column_type=False,
```

Comment:
We need to set `check_column_type=False` because in Pandas > 2.0 set empty data frame to RangeIndex and that's not the case when you delete a column. Hence it will fail on case 10-12